### PR TITLE
RDISCROWD-3853 Show row number upon csv import ASCII error

### PR DESCRIPTION
**Pull request by kbecker42**
- Added the row number to the error message when a csv file fails to import due to a non-ASCII character unicode error.

## Screenshots

cap1

### CODE REVIEW

File: pybossa/importers/csv.py [hunk 1]
```diff
@@ -354,6 +355,18 @@ def __init__(self, **form_data):
...
 
+    def _find_non_ascii_row(self, file_name):
+            """ Returns the first row number and prior row content in the file containing a non-ASCII (utf-8) character."""
+            with open(file_name, 'rb') as src:
+                line_number = 1
+                for line in src:
+                    try:
+                        s = codecs.decode(line, encoding='utf-8')
+                    except UnicodeDecodeError as ex:
+                        return (line_number, s)
```
[the remark this answers is not in the document]
Comment (dchhabda) [reply pybossa/importers/csv.py:366]: With exception raised, `s` won't be available and line 366 would result into another exception suppressing the real exception caused by line 364. Also, logging exception would be useful before line 366.

**Review by dchhabda — CHANGES_REQUESTED** [review]
Please add tests to improve coverage. Thanks!

**Comment by kbecker42** [comment]
Fixed via https://github.com/bloomberg/pybossa/pull/749/commits/bcfb312c560b0b67332157f2678c224f36eb84a8

File: pybossa/importers/csv.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbecker42) [reply pybossa/importers/csv.py]: Fixed via https://github.com/bloomberg/pybossa/pull/749/commits/bcfb312c560b0b67332157f2678c224f36eb84a8

**Pull request merged**